ios/WatchBetterRail Extension/Models/RouteModel.swift

```diff
@@ -45,6 +45,7 @@ struct Train: Decodable, Identifiable, Encodable {
   var stationImage: String? { getStationById(orignStation)?.image }
   var destinationStationImage: String? { getStationById(destinationStation)?.image }
 
+  let delay: Int
```

Comment:
Why was it needed?

Reply:
We already send the delay in the route from RN to Swift but the key wasn't mapped.